src/wallet/crypter.h

```diff
@@ -37,11 +37,9 @@ class CMasterKey
     std::vector<unsigned char> vchCryptedKey;
     std::vector<unsigned char> vchSalt;
     //! 0 = EVP_sha512()
-    //! 1 = scrypt()
     unsigned int nDerivationMethod;
     unsigned int nDeriveIterations;
-    //! Use this for more parameters to key derivation,
-    //! such as the various parameters to scrypt
+    //! Use this for more parameters to key derivation (currently unused)
     std::vector<unsigned char> vchOtherDerivationParameters;
```

Comment:
In 62c209f50d9c33:
What about inlining this into the (un)serialization method?
If it is not used, we don't need to keep the field in memory.